app/scripts/migrations/126.ts

```diff
@@ -0,0 +1,42 @@
+import { hasProperty } from '@metamask/utils';
+import { cloneDeep } from 'lodash';
+
+type VersionedData = {
+  meta: { version: number };
+  data: Record<string, unknown>;
+};
+
+export type PPOMController = {
+  versionInfo?: Record<string, unknown>[];
+  versionFileETag?: string;
+};
+
+export const version = 126;
+
+/**
+ * This migration deletes field versionFileETag from PPOMController.
+ *
+ * @param originalVersionedData - Versioned MetaMask extension state, exactly
+ * what we persist to dist.
+ * @param originalVersionedData.meta - State metadata.
+ * @param originalVersionedData.meta.version - The current state version.
+ * @param originalVersionedData.data - The persisted MetaMask state, keyed by
+ * controller.
+ * @returns Updated versioned MetaMask extension state.
+ */
+export async function migrate(
+  originalVersionedData: VersionedData,
+): Promise<VersionedData> {
+  const versionedData = cloneDeep(originalVersionedData);
+  versionedData.meta.version = version;
+  transformState(versionedData.data);
+  return versionedData;
+}
+
+function transformState(state: Record<string, unknown>) {
+  if (!state.PPOMController) {
+    return state;
+  }
+  delete (state.PPOMController as PPOMController).versionFileETag;
```

Comment:
For audit sake, my understanding is that `delete` is happy if the property already doesn't exist.